src/main/java/eu/bbmri_eric/negotiator/plugins/resourcesync/schedulers/JobEventScheduler.java

```diff
@@ -0,0 +1,33 @@
+package eu.bbmri_eric.negotiator.plugins.resourcesync.schedulers;
+
+import eu.bbmri_eric.negotiator.service.DiscoverySynchronizationJobService;
+import lombok.extern.apachecommons.CommonsLog;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@CommonsLog
+public class JobEventScheduler {
+
+  @Autowired DiscoverySynchronizationJobService discoverySynchronizationJobService;
+
+  @Value("${synchronization.discoveryServiceId}")
+  private String synchronizationDiscoveryServiceId;
+
+  @Scheduled(cron = "${synchronization.frequency}") // Cron expression for running every minute
```

Comment:
this is why I'm not too fond of code comments: the comment is clearly outdated and therefore confusing, making the code even less readable than if the comment wasn't there

Reply:
fixed in 53131252c6f43525c044b84ec3b7c262b19835ed